src/containers/add-attachments/AddAttachments.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  root: {
+    width: '65vw',
+    padding: '20px'
```

Comment:
As I understand, for now we use `p` instead of `padding`, but that's just a matter of choice.

Reply:
done